### ISSUE TITLE
first element in dropdown does not have a hitbox in iOS accessibility

### ISSUE THREAD

**Issue by Davidbriar** [JetBrains/compose-multiplatform#4586]
**Describe the bug**

Using accessibility on iOS, unable to select the first element in a dropdown. No hitbox is available for this element.
(Note: The hotbox is available when the dropdown is initially shown, but disappears subsequently.)

Using accessibility inspector to view hit boxes

I am using accessibility to do instrumented tests on the iOS simulator

**Affected platforms**
- iOS

**Versions**
- Kotlin version*:  1.9.22
- Compose Multiplatform version*: 1.6.10-Dev1561
- OS version(s)* (required for Desktop and iOS issues): 14
- OS architecture (x86 or arm64): arm64
- Device (model or simulator for iOS issues): Xcode simulator
- JDK (for desktop issues): 

**To Reproduce**
Steps and/or the code snippet to reproduce the behavior:
1. Go to DropBox and select to show dropdown
2. Unable to see hitbox for first element. 

repo: 
https://github.com/PythiaTec/ComposeDemo

**Expected behavior**
Expect to be able to select the first element in the dropdown using accessibility.

**Screenshots**
see attached
<img width="376" alt="Screenshot 2024-04-07 at 2 49 16 PM" src="https://github.com/JetBrains/compose-multiplatform/assets/141596630/2cf0a320-2485-4ed3-8fb1-756771554df4">
<img width="376" alt="Screenshot 2024-04-07 at 2 49 29 PM" src="https://github.com/JetBrains/compose-multiplatform/assets/141596630/e9a2298d-ff6d-4ab9-aa0c-c3084e528f49">
<img width="376" alt="Screenshot 2024-04-07 at 2 50 13 PM" src="https://github.com/JetBrains/compose-multiplatform/assets/141596630/1944d756-0d22-440f-92e2-05d6337f0710">

**Additional context**
I also noticed on the demo, when the picture is shown, the hotboxes do not adjust. (Ignore this)

**Comment by elijah-semyonov** [comment]
I indeed reproduced this bug in Accessibility Inspector, but I have no idea what to do with it.
On real iOS device the DropDown elements are achievable by both structural navigation (swipes) and hit-test interaction (holding a finger to scan the UI underneath).

The bounds calculation bug was fixed, expect it on the next release.

**Comment by Davidbriar** [comment]
Thanks for looking into this. I was trying to set up some instrumented xctest, similar to what I have on the android side, I’ll play around with it, maybe it’s a simulator thing.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4586">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
